.github/workflows/generate-command-reference.yml

```diff
@@ -0,0 +1,29 @@
+name: Automatically generate command reference topics
+on:
+  push:
+  workflow_dispatch:
+
+env:
+  AMC_COMMAND_REFERENCE_DIR: "./reference/cmd-ref/amc-command-reference"
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-24.04
+    steps:
+    - name: Setup Anbox Cloud
+      uses: canonical/anbox-cloud-github-action@main
+      with:
+        ubuntu-pro-token: ${{ secrets.UBUNTU_PRO_TOKEN }}
+    - name: Checkout repo
+      uses: actions/checkout@v4
+    - name: Delete existing files
+      run: rm -rf $AMC_COMMAND_REFERENCE_DIR/*
+    - name: Generate AMC command reference documentation
+      run: amc generate documentation -f markdown -d $AMC_COMMAND_REFERENCE_DIR
```

Comment:
If we really only need `amc ` you could also do `snap install amc` instead of using the appliance which should be a lot faster.

Reply:
We need this action to do the same for appliance commands as well. So I will need the full appliance